scintillometry/tests/test_io_psrfits_write.py

```diff
@@ -0,0 +1,30 @@
+# Licensed under the GPLv3 - see LICENSE
```

Comment:
I'd add writing tests to the regular psrfits test file

Reply:
Could you explain this comment more? Do you mean a real pulse profile PSRFITS file?

Reply:
I mean to add tests to `test_io_psrfits.py` rather than create a new test file.